physics/dynamic_frame_test.cpp

```diff
@@ -400,6 +401,103 @@ TEST_F(DynamicFrameTest, CentrifugalAcceleration) {
                            0));
 }
 
+// A frame initially nonrotating and in uniformly accelerated rotation around
+// the origin.  The test point is on the x axis.  The acceleration is purely due
+// to Euler.  The motion elements that don't have specific values have no effect
+// on the acceleration.
+TEST_F(DynamicFrameTest, EulerAcceleration) {
+  Instant const t0;
+
+  EXPECT_CALL(mock_frame_, MotionOfThisFrame(_))
+      .WillRepeatedly(Invoke([t0](Instant const& t) {
+        AngularAcceleration const ωʹ = 10 * Radian / Second / Second;
+        Bivector<AngularAcceleration, Inertial> const
+            angular_acceleration_of_to_frame({0 * Radian / Second / Second,
+                                              0 * Radian / Second / Second,
+                                              ωʹ});
+        Rotation<Inertial, Rotating> const rotation(
+            ωʹ * Pow<2>(t - t0) / 2,
+            angular_acceleration_of_to_frame,
+            DefinesFrame<Rotating>{});
+        RigidTransformation<Inertial, Rotating> const
+            rigid_transformation(
+                /*from_origin=*/Inertial::origin,
+                /*to_origin=*/Rotating::origin,
```

Comment:
On second thought, it is probably a bad idea to conflate these origins, since `Rotating::origin` is involved in the computation (it is the axis), and `Inertial::origin` should not be, but could easily be used.
Since this has no effect on the expectations, use a general position for all the tests.